plugins/source/digitalocean/docs/digitalocean_account.md

```diff
@@ -0,0 +1,18 @@
+
```

Comment:
We shouldn't have the generated docs files committed to the repo anymore--they will be generated as part of the website build.

(We do need to keep the old docs under `tables/` for now though, just to not break the current website until v2 is fully released)